nntrainer/layers/layer.cpp

```diff
@@ -84,6 +100,74 @@ void Layer::copy(std::shared_ptr<Layer> l) {
   this->num_outputs = l->num_outputs;
 }
 
+sharedConstTensors Layer::forwarding_with_val(sharedConstTensors input,
```

Comment:
what is the difference between input and in?

Reply:
my bad, I think in must be deleted. done